0_json_functions.R

```diff
@@ -354,41 +354,32 @@ export_audit_invalid_data <- function(portfolio_total_, export_path_full) {
 }
 
 export_audit_textvar_json <- function(portfolio_total_, export_path_full) {
-  portfolio_total_ <- portfolio_total_ %>% select("value_usd", "asset_type", "has_valid_input")
-  total_v <- sum(portfolio_total_$value_usd)
-  included <- portfolio_total_ %>% subset(has_valid_input == TRUE)
-  included_v <- sum(included$value_usd)
-  bonds <- portfolio_total_ %>% subset(asset_type == "Bonds")
-  bonds_v <- sum(bonds$value_usd)
-  equity <- portfolio_total_ %>% subset(asset_type == "Equity")
-  equity_v <- sum(equity$value_usd)
+  total <- round(sum(portfolio_total_$value_usd, na.rm = TRUE))
 
-  assertthat::are_equal(included_v <= total_v, TRUE)
-  assertthat::are_equal(bonds_v + equity_v - included_v < 1, TRUE)
-
-  total_v <- round(total_v)
-  total_v <- as.character(total_v)
-  included_v <- round(included_v)
-  bonds_v <- round(bonds_v)
-  equity_v <- round(equity_v)
-
-  keys <- c("\"total\"", "\"included\"", "\"bonds\"", "\"equity\"")
-  values <- c(total_v, included_v, bonds_v, equity_v)
-
-
-  json_head <- "{"
-  json_tail <- "}"
+  included <-
+    portfolio_total_ %>%
+    filter(has_valid_input == TRUE) %>%
```

Comment:
Non-blocking: for future work, can we get in the habit of calling it as `dplyr::filter()`?

Reply:
wrt to `jsonlite`: I verified that the Docker has `jsonlite` installed, at least according to [the Docker file that creates the image that it is based on](https://github.com/2DegreesInvesting/docker/blob/9efe76454b91a45f47e32da5d5d7d440d6a8a644/r-packages/Dockerfile#L28). As far as it being in `DESCRIPTION`... that's kinda a Mauro thing because this repo is not an R package so I basically ignore that stuff.

Reply:
wrt to `dplyr::filter()`: sure, fair, I'll make that change

Reply:
> wrt to `jsonlite`: I verified that the Docker has `jsonlite` installed, at least according to [the Docker file that creates the image that it is based on](https://github.com/2DegreesInvesting/docker/blob/9efe76454b91a45f47e32da5d5d7d440d6a8a644/r-packages/Dockerfile#L28). As far as it being in `DESCRIPTION`... that's kinda a Mauro thing because this repo is not an R package so I basically ignore that stuff.

@maurolepore, FYI

Reply:
> wrt to `dplyr::filter()`: sure, fair, I'll make that change

switching to `dplyr::filter()` isn't a huge issue, but i've occasionally run into problems in the past where R reverts back to `base:filter()` in certain situations. It's a pain of a bug to reproduce, but it's pretty easy to avoid.

Reply:
indeed